.gitignore

```diff
@@ -0,0 +1 @@
+node-modules
```

Comment:
- [ ] As stated in the project requirement, there should be a test.md file in your root directory with a text `this file should be ignored by git` and the file should be ignored in your .gitignore file. Kindly implement this so as to follow the project requirement strictly